index.html

```diff
@@ -134,7 +134,6 @@
                     records: 10,
                     // maximum duration between start and end bounds
                     // for utc-based time systems this is in milliseconds
-                    limit: ONE_DAY
```

Comment:
i might leave this commented so we know its possible to enforce a limit if desired